main.lua

```diff
@@ -163,7 +174,11 @@ function thm:updateData(msg)
 end
 
 function thm:firstUpper(str)
-	return string.upper(str:sub(1,1))..string.lower(str:sub(2))
+	if not str then
+		return string.upper(str:sub(1,1))..string.lower(str:sub(2))
+	else
+		return
+	end

```

Comment:
This... doesn't really have any benefit, this should never receive a null value (the fix should be in the main function to never call if there's no value)
Otherwise it crashes later when attempting to blacklist the null value :)

Reply:
Yea this was an attempt to fix an error that I knew would occur because the current addon throws a wobbly when you try and parse nothing. 
The line it errors on is here when trying to parse the nothing valve of "/thm bl" so I tried to bodge job fix it where the problem was but you're right that it should be fixed back in the main function.

Reply:
I don't know how to change or remove this commit (section of code)

Reply:
It's done locally, as a new commit :)
Or if it's ok with you I can accept the pull and revert it with another commit :)

Reply:
I think that last option might be the fastest!